app/com/gitpitch/utils/YAMLOptions.java

```diff
@@ -164,6 +164,16 @@ public String fetchImageBg(PitchParams pp) {
         }
     }
 
+    public String fetchImageBgSize(PitchParams pp) {
+        String bgSize = _yProps.get(IMAGE_BG_SIZE_OPTION);
+        if (bgSize == null) {
+            return DEFAULT_BG_SIZE;
+        } else {
+            return bgSize;
+        }
+    }
+
+
```

Comment:
Looks like one extra empty line has crept into the file here. Can you please remove.